package.json

```diff
@@ -255,8 +255,10 @@
     "replace-in-files-cli": "^2.2.0",
     "rimraf": "^5.0.1",
     "sass": "~1.79.0",
+    "sass-embedded": "~1.79.0",
     "sass-loader": "^14.0.0",
     "semver": "^7.5.2",
+    "source-map-js": "^1.2.1",
```

Comment:
Maybe not needed, I had a temporary issue with nx cache, so I'll check if this can be reverted.